src/test/getarg_tests.cpp

```diff
@@ -144,6 +145,11 @@ BOOST_AUTO_TEST_CASE(intarg)
     BOOST_CHECK_EQUAL(m_local_args.GetIntArg("-foo", 11), 0);
     BOOST_CHECK_EQUAL(m_local_args.GetIntArg("-bar", 11), 0);
 
+    // Check under-/overflow behavior.
+    ResetArgs("-foo=-9223372036854775809 -bar=9223372036854775808");
```

Comment:
This is not overflowing. It is equal to the min/max?

Reply:
```cpp
10:27:37 james@slug james/tmp % cat int64max.cpp
#include <limits>
#include <iostream>
#include <cinttypes>

int main() {
    std::cout << std::numeric_limits<int64_t>::max() << '\n';
}
10:27:44 james@slug james/tmp % ./a.out
9223372036854775807
```